### PR TITLE
Added WireMock to the Hacktoberfest Swag List (#739)

### PR DESCRIPTION
**Pull request by biplobsd**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -355,6 +355,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: [Eligible projects and issues](https://github.com/search?q=org%3Aturbot+topic%3Ahacktoberfest&type=repositories).
 - **Notes**: Read more details in the [announcement post](https://turbot.com/blog/2023/10/hacktoberfest-2023) and join our [Turbot community](https://turbot.com/community/join) `#contributors` channel with any questions.
 
+### W
+
+#### **WireMock**
+
+- **Swag**: T-Shirts, Stickers, Special swag.
+- **Requirements**:
+    - All projects in the official WireMock organization on GitHub.
```
Comment (vimode) [inline docs/list.md:364]: These are *where to contribute*, the requirements section needs the list of things one needs to do to be eligible to win the swag(s).

You could look into the other submissions at [hacktoberfestswaglist](https://hacktoberfestswaglist.com/list/) or check the [contributing guide](docs/contributing.md) to get help with this section.

File: docs/list.md [hunk 2]
```diff
@@ -355,6 +355,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+    - [WireMock Ecosystem](https://github.com/wiremock/ecosystem) projects marked with a `hacktoberfest` topic by their maintainers.
+    - Judge contributions by the most active contributors, quality and value.
+- **How to sign up**: Sign up for [registration form](https://docs.google.com/forms/d/e/1FAIpQLSfC_8mX8lpQNhksqQGUhsQ0zTvRzMC5YzufVVbVkPksxouLFw/viewform)
```
Comment (crweiner) [inline docs/list.md:367]:
```suggestion
- **How to sign up**: Sign up for [registration form](https://docs.google.com/forms/d/e/1FAIpQLSeBiJSYV4PNFYOH1Jo9PSheOYKxqV2NJE3LELL9a_fYq4fpfw/viewform?usp=sf_link)
```
I believe you may have the wrong Google Form link here, @biplobsd. The one that you have links to the "Treblle Hacktoberfest Hackathon." Is this a separate organization that we should also have on the Swag List?

File: docs/list.md (hunk 1 above)
Comment (biplobsd) [reply docs/list.md]: Hey @vimode, I removed that first line. Now is it okay?

File: docs/list.md (hunk 2 above)
Comment (vimode) [reply docs/list.md]: @crweiner I am unsure if Treblle's hacktoberfest hackathon would be counted towards the DO hacktoberfest event PR count. If thats OK, then a must add. That hackathon looks amazing, the landing page itself is so inspiring! 🎃

Comment (crweiner) [reply docs/list.md]: > That hackathon looks amazing, the landing page itself is so inspiring! 🎃

[I just pulled it up myself and yes, that does look amazing!](https://hacked.treblle.com) 😍 

[According to their blog post](https://blog.treblle.com/treblle-hacktoberfest/), Treblle will link to your newly-created SDK, which will be in the submitter's own GitHub repo. You _can_ use your own projects to count towards Hacktoberfest, as long as the repo is opted-in and you are still making meaningful PRs.

With that in mind, I say we can add it, as long as we note that you must tag the repo for Hacktoberfest yourself. 🎃

Comment (crweiner) [reply docs/list.md]: Forgot to ping you on the above, @vimode! ⬆️

**Comment by crweiner** [comment]
Thank you very much for your PR to add WireMock to the Hacktoberfet Swag List, @biplobsd! I appreciate you making the corrections that @vimode  and I pointed out. I hope you're having a great Hacktoberfest! 🎉 

thank you

**Comment by biplobsd** [comment]
@crweiner , @vimode Thanks for merged🚀

File: docs/list.md (hunk 2 above)
Comment (vimode) [reply docs/list.md]: @crweiner 

> You can use your own projects to count towards Hacktoberfest, as long as the repo is opted-in and you are still making meaningful PRs.

Wasn't aware of this rule. Thats great. Would you be adding it in your next merge or we could create a new issue?

**Pull request merged**